saltgui/static/scripts/config.js

```diff
@@ -0,0 +1,4 @@
+const config = {
+  // API_URL: '/api'
```

Comment:
or 'ht<i></i>tp://hostname:port'

Reply:
The problem here is that we can't use fully independent salt-api as mentioned in the main readme.